lib/libutee/include/tee_api_defines.h

```diff
@@ -45,6 +45,7 @@
 #define TEE_ERROR_SIGNATURE_INVALID       0xFFFF3072
 #define TEE_ERROR_TIME_NOT_SET            0xFFFF5000
 #define TEE_ERROR_TIME_NEEDS_RESET        0xFFFF5001
+#define TEE_ERROR_NOT_CONFIGURED          0xFFFF9000
```

Comment:
As far as I know this isn't an error code found in any GP specification. If it's an implementation specific extension then it should go in the range 0x80000000 – 0x8FFFFFFF.

In this case it looks like it's just a TA that need to return some error code, in that case you can define it as whatever except `0` and preferably avoid known error code. The define should go into the TAs exported header, in this case `ta_services/keymaster/include/keymaster_ta.h`

Reply:
Ok. Will move to `keymaster_ta.h`.